README.md

```diff
@@ -65,7 +65,22 @@ Start indexing documents simply by creating or saving them:
 Artist.create(name: 'Frank Stella', keywords: ['art', 'minimalism'])
 ```
 
-Stella adds `after_save` and `after_destroy` callbacks for inline indexing, override these callbacks (namely `#es_index` and `#es_delete`) if you'd like to do your indexing in a background process.
+Stella adds `after_save` and `after_destroy` callbacks for inline indexing, override these callbacks if you'd like to do your indexing in a background process. For example:
+
+```ruby
+class Artist < ActiveRecord::Base
+  include Stella::Searchable
+
+  # disable stella inline callbacks
+  skip_callback(:save, :after, :es_index)
+  skip_callback(:destroy, :after, :es_delete)
+
+  # declare your own
+  after_save :delay_es_index
+  after_destroy :delay_es_delete
```

Comment:
(This has the benefit of using existing code to accomplish what it needs, so on 2nd thought don't feel any pressure to implement something custom instead. Just thinking out loud and imagining what the ideal developer experience is.)